### DISCUSSION TITLE
[destination-bigquery] error during config `malformed JSON at line 1 column 1 path $`

### DISCUSSION THREAD

**Discussion by rubenatterbury** [airbytehq/airbyte#34097] [Connector Questions]
I've been experiencing a lot issues in setting up BigQuery with Airbyte. I have to use Airbyte because we have a strict VPN control over our database, so could only get it to work when I bought a server and installed airbyte. The issue now I want to load data from Microsoft SQL Server to BigQuery.

I'm not well versed in BigQuery but have followed the million steps to try and activate every API and set up the many keys and account details I need for the sync (BigQuery as destination). Now I'm getting this error:
"Configuration check failed
com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 1 path $ "
Google can't help with the issue. Can you please assist me? I'm quite desperate at this stage to get it to work.

When I try and e-mail support to team@airbyte.io the e-mail bounces? And there's not related topic on the help center. Any help would be greatly appreciated.

**Comment by marcosmarxm** [comment]
This looks you're trying to paste a malformated service account to be used by the config.
Make sure the service account is a valid JSON, you can use online tools or your IDE to validate it.
Also can be very useful to share what version of the Bigquery connector are you using.

[a comment by an automated account is not shown]
